amr-wind/wind_energy/actuator/wing/fixed_wing_ops.H

```diff
@@ -161,15 +212,17 @@ struct InitDataOp<FixedWing, ActSrcLine>
                 meta.span_locs, meta.chord_inp, wx, meta.chord);
             // clang-format off
             meta.epsilon_chord = {
-                meta.epsilon_chord.x(), meta.epsilon_chord.z(),
-                meta.epsilon_chord.y()};
+                meta.epsilon_chord.x(), meta.epsilon_chord.y(),
+                meta.epsilon_chord.z()};
```

Comment:
@ET14 I think this is why you were seeing that mismatch in the epsilon components. The code was switching the place of the y and z components when it relied on epsilon_chord. I have to check this, though.

Reply:
I was wrong about this change, but I was close. This reordering operation is necessary, but it was not being performed for eps_inp. Fixed.